server/client-shared/src/main/scala/cool/graph/client/requestPipeline/FunctionExecutor.scala

```diff
@@ -121,7 +129,7 @@ class FunctionExecutor(implicit val inj: Injector) extends Injectable {
           logsPublisher.publish(renderLogPayload("FAILURE", Map("error" -> s"Couldn't parse response: $badBody. Error message: $parseError")))
 
         case Success(Bad(FunctionReturnedBadStatus(statusCode, rawResponse))) =>
-          logsPublisher.publish(renderLogPayload("FAILURE", Map("error" -> rawResponse))) //Function returned invalid status code: $statusCode. Raw body:
+          logsPublisher.publish(renderLogPayload("FAILURE", Map("error" -> s"Function returned invalid status code: $statusCode. Raw body: $rawResponse")))
```

Comment:
This is reverting an intentional change, what is the reasoning for changing it back?